### PR TITLE
Fix code style issues in python files in scripts dir and add files to codestyle checking

### PR DESCRIPTION
**Pull request by stefanberger**
Fix code style issues in python files in scripts directory and add the fixed files to codestyle checking.

### CODE REVIEW

File: scripts/convert_config.py [hunk 1]
```diff
@@ -173,17 +171,17 @@ def process_mapping(old_config, templates, mapping_file, debug=False):
     configuration dictionary
     """
 
-    with open(mapping_file, "r") as f:
+    with open(mapping_file, "r", encoding="utf-8") as f:
         try:
             mapping = json.loads(f.read())
         except Exception as e:
-            raise Exception(f"Could not load mapping file {mapping_file}: {e}")
+            raise Exception(f"Could not load mapping file {mapping_file}: {e}") from e
```
Comment (aplanas) [inline scripts/convert_config.py:178]: That would make safe to remove `{e}` from the string?

**Review by aplanas — APPROVED** [review]
LGTM

File: scripts/convert_config.py (hunk 1 above)
Comment (stefanberger) [reply scripts/convert_config.py:178]: Not so clear to me...

```
#!/usr/bin/env python3

try:
    raise Exception("Test")
except Exception as e:
    raise Exception(f"Got {e}") from e
```

Result: 
```
  File "/home/stefanb/keylime/./test.py", line 4, in <module>
    raise Exception("Test")
Exception: Test

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/stefanb/keylime/./test.py", line 6, in <module>
    raise Exception(f"Got {e}") from e
Exception: Got Test
```

Without `from e`:

```
#!/usr/bin/env python3

try:
    raise Exception("Test")
except Exception as e:
    raise Exception(f"Got {e}")
```

Result:
```
Traceback (most recent call last):
  File "/home/stefanb/keylime/./test.py", line 4, in <module>
    raise Exception("Test")
Exception: Test

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stefanb/keylime/./test.py", line 6, in <module>
    raise Exception(f"Got {e}")
Exception: Got Test
```

**Comment by THS-on** [comment]
@stefanberger can you add those scripts to the `make check` command so that don't reintroduce those issues again? Otherwise LGTM.

**Comment by stefanberger** [comment]
> @stefanberger can you add those scripts to the `make check` command so that don't reintroduce those issues again? Otherwise LGTM.

Already done via patches starting with title `codestyle:`.

**Comment by THS-on** [comment]
> Already done via patches starting with title codestyle:.

Yep, I somehow skipped over that sorry.

**Review by THS-on — APPROVED** [review]

**Pull request merged**